CPT331.Data.Migration/Migrations/M0001.cs

```diff
@@ -15,50 +15,45 @@ public override void Down()
 		{
 			//	Nothing to see here
 		}
-
+        
 		public override void Up()
 		{
+            // Database Schemas
 			Create.Schema("Crime");
 			Create.Schema("Location");
 
-			Create.Table("Crime")
-				.InSchema("Crime")
-					.WithColumn("ID")
-						.AsInt32()
-						.NotNullable()
-						.Identity()
-						.PrimaryKey()
-					.WithColumn("LocalGovernmentAreaID")
-						.AsInt32()
-						.NotNullable()
-					.WithColumn("OffenceID")
-						.AsInt32()
-						.NotNullable()
-					.WithColumn("Count")
-						.AsInt32()
-						.NotNullable()
-					.WithColumn("Month")
-						.AsInt32()
-						.NotNullable()
-					.WithColumn("Year")
-						.AsInt32()
-						.NotNullable()
-					.WithColumn("IsDeleted")
-						.AsBoolean()
-						.NotNullable()
-						.WithDefaultValue(false)
-					.WithColumn("IsVisible")
-						.AsBoolean()
-						.NotNullable()
-						.WithDefaultValue(true)
-					.WithColumn("DateCreatedUtc")
-						.AsDateTime()
-						.NotNullable()
-						.WithDefault(SystemMethods.CurrentUTCDateTime)
-					.WithColumn("DateUpdatedUtc")
-						.AsDateTime()
-						.NotNullable()
-						.WithDefault(SystemMethods.CurrentUTCDateTime);
+            //	Tables
+            Create.Table("State")
```

Comment:
The order of tables in the migration code was changed to compensate for the foreign keys.

1. State (_Fluent_)
2. LocalGovernmentArea (_Script_)
3. OffenseCategory (_Fluent_)
4. Offense (_Fluent_)
5. Crime (_Fluent_)